docs/architecture/tls13-experimental.md

```diff
@@ -133,17 +133,25 @@ MVP definition
 
   (1) This is just for comparison.
 
-  (2) The MVP sends one shared secret corresponding to the configured preferred
-      group. The preferred group is the group of the first curve in the list of
-      allowed curves as defined by the configuration. The allowed curves are
-      by default ordered as follow: `secp256r1`, `x25519`, `secp384r1`
-      and finally `secp521r1`. This default order is aligned with the
-      list of mandatory-to-implement groups (in absence of an application
-      profile standard specifying otherwise) defined in section 9.1 of the
-      specification. The list of allowed curves can be changed through the
-      `mbedtls_ssl_conf_curves()` API.
-
-  (3) The MVP proposes only TLS 1.3 and does not support version negociation.
+  (2) The MVP sends only one shared secret corresponding to the configured
+      preferred group. This could, however end up with connection failure if the
+      server does not support our preferred curve, as we have yet to implement
+      HelloRetryRequest. The preferred group is the group of the first curve in
+      the list of allowed curves as defined by the configuration. The list of
+      mandatory-to-implement groups (in absence of an application profile
+      standard specifying otherwise) as defined in section 9.1 of the
+      specification gives the preferred order as follows: `secp256r1`, `x25519`,
+      `secp384r1` and finally `secp521r1`. If we could therefore fix the use of
```

Comment:
At this point I do not think it is clear what "fix" is referring to as the actual default order has not been introduced yet.